docs/devguide/run_time_configuration.rst

```diff
@@ -1103,22 +1106,22 @@ Those properties, along with options specific to OpenDDS's RTPS discovery implem
   .. prop:: ResendPeriod=<sec>
     :default: ``30``
 
-    The number of seconds that a process waits between the announcement of participants (see :omgspec:`rtps:8.5.3`).
+    The number of seconds that a process waits between the :ref:`SPDP participants announcements <spdp>`.
 
   .. prop:: MinResendDelay=<msec>
     :default: ``100``
 
-    The minimum time in milliseconds between participant announcements.
+    The minimum time in milliseconds between :ref:`SPDP participants announcements <spdp>`.
 
   .. prop:: QuickResendRatio=<frac>
     :default: ``0.1``
 
-    Tuning parameter that configures local SPDP resends as a fraction of the resend period.
+    Tuning parameter that configures local :ref:`SPDP participants announcement <spdp>` resends as a fraction of the resend period.
```

Comment:
```suggestion
    Tuning parameter that configures local :ref:`SPDP participant announcement <spdp>` resends as a fraction of the resend period.
```

Reply:
Maybe consider incorporating https://github.com/OpenDDS/OpenDDS/discussions/4684#discussioncomment-9731942

Reply:
Sure, but I want to make sure I'm understanding what this is doing correctly:

> The ResendPeriod then gradually increases back to its original value. 

I was curious about what you exactly meant by "gradually". I looked at the code and did a simple test and looks like it just does the quick resend and then goes back to the normal resend period.